google-cloud-speech/lib/google/cloud/speech.rb

```diff
@@ -21,7 +21,7 @@ module Cloud
     # rubocop:disable LineLength
 
     ##
-    # # Ruby Client for Cloud Speech API ([Alpha](https://github.com/googleapis/google-cloud-ruby#versioning))
+    # # Ruby Client for Cloud Speech API ([Beta](https://github.com/googleapis/google-cloud-ruby#versioning))
```

Comment:
Shouldn’t this change happen in synth.py?

Reply:
I don't know. @TheRoyalTnetennba @dazuma what do you think?

Reply:
Same for `v1.rb` and` v1p1beta1.rb` I guess.

Reply:
Ideally it would happen in synth. Same for google-cloud-speech/README.md.

Reply:
These get configured in the gapic config files. We can (and I think we should) fix them upstream. Happy to OK this PR as is, and I'll submit a fix internally to the gapic configs so the next autosynth doesn't revert it.

Reply:
I updated this PR to only change the top-level README. (PR #2838 now updates the Speech README well as the `speech.rb`, `v1.rb`, and `v1p1beta1.rb` docs.)

Reply:
Aaaand it looks like we're not moving forward on speech GA for now after all. (There was a miscommunication). So we're rolling back the gapic config to alpha, and we should drop this PR.